resources/dicts/events/injury/general/general.json

```diff
@@ -795,5 +795,58 @@
         "cat_skill": null,
         "other_cat_trait": null,
         "other_cat_skill": null
+    },
+        {
+        "injury": "headache",
+        "camp": "any",
+        "tags": [
+            "Newleaf",
+            "Greenleaf",
+            "Leaf-fall",
+            "Leaf-bare"
+        ],
+        "event_text": "Chasing a mouse through a dense area, m_c accidentally collides with a low-hanging branch, leaving them to return to camp empty-pawed and with splitting head pain.",
```

Comment:
I really like this event but would like to make it forest specific because it's got nice environmental description

Reply:
👍